docs/legacy/configuration-agent-config.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[configure-agent-config]]
+== Configure APM agent configuration
+
+++++
+<titleabbrev>APM agent configuration</titleabbrev>
+++++
+
+IMPORTANT: {deprecation-notice-config}
+
+APM agent configuration allows you to fine-tune your APM agents from within the APM app.
+Changes are automatically propagated to your APM agents, so there's no need to redeploy your applications.
+
+To learn more about this feature, see {kibana-ref}/agent-configuration.html[APM agent configuration].
+
+Here's a sample configuration:
+
+[source,yaml]
+----
+apm-server.agent.config.cache.expiration: 45s
+apm-server.agent.config.elasticsearch.api_key: TiNAGG4BaaMdaH1tRfuU:KnR6yE41RrSowb0kQ0HWoA
+----
+
+[float]
+=== APM agent configuration options
+
+You can specify the following options in the `apm-server.agent.config` section of the
++{beatname_lc}.yml+ config file:
+
+[float]
+[[agent-config-cache]]
+==== `apm-server.agent.config.cache.expiration`
+
+When using APM agent configuration, information fetched from {es} will be cached in memory for some time.
+Specify the cache expiration time via this setting. Defaults to `30s` (30 seconds).
+
+[float]
+[[agnent-config-authentication]]
+=== Authentication credentials
+
+For APM Server legacy users,
+APM agent configuration is automatically fetched from {es} using the `output.elasticsearch`
+configuration. Use these authentication configuration variables to fine-tune permission access.
+
+When using a Fleet-managed APM Server or Elastic Agent standalone-managed APM Server,
```

Comment:
I don't know enough about fleet-managed and elastic agent standalone-managed setup to confirm this. I believe fleet should work out of the box, not too sure about elastic agent standalone. @simitt could you double check if this statement is correct?

Reply:
spot on - for Fleet the configuration is automatically injected via the config that is passed down. 
For standalone Elastic Agent, the same information as for APM Server legacy applies - is `output.elasticsearch` credentials are provided and privileged enough, they'll be used, otherwise users have to configure specific credentials.

Reply:
Thanks! Will fix.